content/rest/_request.textile

```diff
@@ -0,0 +1,21 @@
+h6(#request).
+  default: request
+
+bq(definition).
+  default: request(String method, String path, Object params, Object body, Object headers, callback("ErrorInfo":/rest/types#error-info err, "HttpPaginatedResponse":/rest/types#http-paginated-response results)))
+
+Makes a REST request to a provided path. This is provided as a convenience for developers who wish to use bleeding edge REST API functionality that is either not documented or is not yet included in the public API, without having to handle authentication, paging, fallback hosts, MsgPack and JSON support, etc. themselves.
+
+h4. Parameters
+
+- method := either @get@ or @post@.<br>__Type: String__
```

Comment:
mostly because currently, PaginatedResource, Resource, Http, and Request currently all only support `get` and `post`, so I know those work. Adding `put` would definitely be possible, but not completely trivial (for one thing we'd need to change how jsonp works in realtime as it currently just uses the presence of a body to decide if get/post (probs just add a `method` param), make sure it works with xhr in old versions of IE, etc. etc.)

So we could certainly do it if we think it's important, but if so we should add it to the spec and decide how we want to prioritise it